airflow/providers/google/cloud/utils/openlineage.py

```diff
@@ -79,3 +92,265 @@ def get_identity_column_lineage_facet(
         }
     )
     return column_lineage_facet
+
+
+@define
+class BigQueryJobRunFacet(BaseFacet):
```

Comment:
I don't have strong opinion on that, maybe @mobuchowski has.
AFAIK e.g. Spark custom facets don't have their JSON Schema specs defined anywhere but I agree this could be stored somewhere. For sure we don't have clear strategy what to do in such cases. If we still want to have JSON Schema - I would put it in Google provider.

Reply:
That's probably something we can discuss in an issue or if there is a consensus - put in the docs. As we do not currently have the jsons for any Airflow specific facet, we can leave it as it is, just wanted to know what's the situation here.

Reply:
Provider specific facet should live in that provider. There's no gain in having it somewhere else if it's not going to be emitted from some other provider. It would only introduce potential versioning problem, and I think we already have enough of them.

Reply:
Should I add the JSON Schema spec file to the provider and point to it in `_get_schema`?

Reply:
Yeah, not sure why not.

Reply:
I moved `utils/openlineage.py` to `openlineage/utils.py` and created JSON Schema spec files in the same directory as well.